README.md

```diff
@@ -1622,6 +1623,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Swapnil Shekhar](https://github.com/Swapnil-programmer7)
   - [Syed Sibteali Baqar](https://github.com/sibteali786)
   - [syedareehaquasar](https://github.com/syedareehaquasar)
+
```

Comment:
Hey @modeware, Could you remove this extra space? So that we can merge your Pull Request!

Reply:
Hey @RakeshSangem , Done. Can you please merge it now.